internal/pkg/scaffold/types.go

```diff
@@ -56,15 +56,15 @@ import (
 type {{.Resource.Kind}}Spec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
-	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io
 }
 
 // {{.Resource.Kind}}Status defines the observed state of {{.Resource.Kind}}
 // +k8s:openapi-gen=true
 type {{.Resource.Kind}}Status struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
-	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io
```

Comment:
```suggestion
	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
```

Reply:
Nice catch, keeping the doc link valuable.